app/src/main/res/values/preference_keys.xml

```diff
@@ -123,4 +123,6 @@
     <string name="preference_key_excluded_wiki_codes_notification">filterNotificationExcludedWikiCodes</string>
     <string name="preference_key_excluded_type_codes_notification">filterNotificationExcludedTypeCodes</string>
     <string name="preference_key_page_notification_tooltip_shown">pageNotificationTooltipShown</string>
+    <string name="preference_key_auto_show_edit_notices">autoShowEditNotices</string>
+    <string name="preference_key_edit_notices_tooltip_shown">editNoticesTooltipShown</string>
```

Comment:
Should we add this to `developer_preferences.xml` as well?

Reply:
Sure!